### ISSUE TITLE
[Slider] Accessibility test failing

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#3160]
### Overview
Failed test on kokoro/bazel:

```
Name: NEW_IPAD_PRO_(12_9-INCH)_9_3
OS: iOS 9.3
Type: iPad Pro (12.9-inch)
2018-03-28 13:38:51,753 Created new simulator 8FE0B5EB-E00B-4146-8CA7-D86F8B9FFE94.
Test Suite 'All tests' started at 2018-03-28 06:38:52.959
Test Suite 'unit_tests_IPAD_PRO_12_9_IN_9_3.xctest' started at 2018-03-28 06:38:52.960
Test Suite 'SliderTests' started at 2018-03-28 06:38:52.960
Test Case '-[SliderTests testAccessibilityDecrement]' started.
Test Case '-[SliderTests testAccessibilityDecrement]' passed (0.069 seconds).
Test Case '-[SliderTests testAccessibilityIncrement]' started.
Test Case '-[SliderTests testAccessibilityIncrement]' passed (0.002 seconds).
Test Case '-[SliderTests testAccessibilityIncrementWithLargerMax]' started.
Test Case '-[SliderTests testAccessibilityIncrementWithLargerMax]' passed (0.001 seconds).
Test Case '-[SliderTests testAccessibilityTraits]' started.
Test Case '-[SliderTests testAccessibilityTraits]' passed (0.001 seconds).
Test Case '-[SliderTests testAccessibilityValue]' started.
components/Slider/tests/unit/SliderTests.m:354: error: -[SliderTests testAccessibilityValue] : (([self.slider accessibilityValue]) equal to (expected)) failed: ("30%") is not equal to ("29%")
```

### Version number
49.0.0

---
https://www.pivotaltracker.com/story/show/156805030
